common/util/FileUtil.cpp

```diff
@@ -0,0 +1,36 @@
+#include "FileUtil.h"
+#include <iostream>
+#include <stdio.h>  /* defines FILENAME_MAX */
+
+#ifdef _WIN32
+  #include <direct.h>
+  #define GetCurrentDir _getcwd
+#else
+  #include <unistd.h>
+  #define GetCurrentDir getcwd
+#endif
+
+std::string FileUtil::GetCurrentWorkingDir() {
+  char buff[FILENAME_MAX];
+  GetCurrentDir(buff, FILENAME_MAX);
+  std::string current_working_dir(buff);
+  return current_working_dir;
+}
+
+std::string FileUtil::get_file_path(std::vector<std::string> input) {
+  std::string currentPath = FileUtil::GetCurrentWorkingDir();  // std::filesystem::current_path();

```

Comment:
Vaser pointed out that we could use `std::filesystem` if we upgraded to C++17, so I did.  Maybe we can use `std::filesystem` here instead, if that's easier.

Reply:
As far as I can tell this would have had the same result as getcwd, so I went ahead and fixed it up the other way that we discussed in the Discord group.